pkg/routeagent/controllers/route/route.go

```diff
@@ -575,24 +601,24 @@ func (r *Controller) handleRemovedPod(obj interface{}) {
 	}
 }
 
-func (r *Controller) cleanRoutes() {
+func (r *Controller) cleanVxSubmarinerRoutes() {
 	link, err := netlink.LinkByName(VxLANIface)
 	if err != nil {
-		klog.Errorf("Error retrieving link by name %s: %v", VxLANIface, err)
+		// if the vx-submariner interface does not exist then there are no routes to be cleaned up, we're fine.
```

Comment:
Looking at the `netlink.LinkByName` code, it will return `netlink.LinkNotFoundError` in that case.  I think we should check for that to be complete.

Reply:
ack, I will do that.